releasenotes/notes/encoder-truncate-2e3b63db1b960561.yaml

```diff
@@ -0,0 +1,4 @@
+fixes:
+  - |
+    tracing: This fix resolves an issue where a very long string as a span attribute would cause that span not to
```

Comment:
should we define what "very long string" means here? since we are only covering the case of "a string larger than the write buffer" ?

should we give an example default size for the max size as well? like, a span attribute larger than 2mb, etc.

wdyt?